assets/l10n/intl_en.arb

```diff
@@ -2532,36 +2532,16 @@
     "noOtherDevicesFound": "No other devices found",
     "fileIsTooBigForServer": "The server reports that the file is too large to be sent.",
     "onlineStatus": "online",
-    "onlineLongTimeAgo": "online long time ago",
-    "onlineMinAgo": "online {min} min ago",
+    "onlineMinAgo": "online {min}m ago",
     "@onlineMinAgo": {
         "placeholders": {
             "min": {}
         }
     },
-    "onlineHourAgo": "online {hour} h {min} min ago",
+    "onlineHourAgo": "online {hour}h ago",
```

Comment:
how about it for other language? Maybe a runtime issue

Reply:
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/48181d55-6eac-4218-ac08-6c75c3228899)
I think onlineHour/minutesAgo aren't translated for other languages